verification/verify_feature.py

```
@@ -0,0 +1,66 @@
+
+def verify_search_tooltip(page: Page):
+    page.on("console", lambda msg: print(f"Browser console: {msg.text}"))
+    page.on("pageerror", lambda err: print(f"Browser error: {err}"))
+
+    # 1. Go to homepage
+    page.goto("http://localhost:5173/")
+
+    # Wait for network idle or load
+    try:
+        page.wait_for_load_state("networkidle", timeout=5000)
+    except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)